lib/utils/id-utils.ts

```diff
@@ -0,0 +1,10 @@
+import { Token } from 'aws-cdk-lib';
+import { v4 as uuid } from 'uuid';
+
+export function uniqueId() : string {
```

Comment:
Could we add a doc on what is the usage of this utility.

Reply:
of course. addressed